src/ui/page/dataverse/dataspace/governance/governance.tsx

```diff
@@ -76,6 +78,17 @@ export const Governance: FC = () => {
     }
   }, [navigate, navigationPath])
 
+  useEffect(() => {
+    if (!dataspaceLabel) {
```

Comment:
You are right. If this request process well but the resource is not found, we must display the 404 error. We are currently using mocked data retrieved with `getResourceDetails`. I do think if we have a server error processing the request to get dataspace (500), this would be a good use case for warning notification.
We can remove it before merging, then add it once the dataspace is connected to the ontology. 
What do you think @lolottetheclash?

Reply:
Yes, the toasts will be for the 500 errors, which we will manage in a second time. 
According to me, currently we only have to manage the 404 👌